docs/development.rst

```diff
@@ -96,6 +96,10 @@ Code style guide
 
 - First and foremost, the linters configured for the project must pass; this generally means following PEP-8 rules,
   as codified by: ``flake8``, ``black``, ``isort``, ``pyupgrade``.
+- The supported Python versions (and the code syntax to use) are listed in the ``tox.ini`` file in the ``tox/envlist``
+  section as ``py3*``. However, there are some files that have to be kept compatible with Python 2.7. They are
+  listed in ``.pre-commit-config.yaml`` under ``repo: https://github.com/asottile/pyupgrade`` under ``hooks/exclude``.
+  Please do not attempt to modernize them to Python 3.x.
```

Comment:
Please add the reason why we need to keep the files Python 2 compatible:
https://github.com/tox-dev/tox/pull/2011#pullrequestreview-632904062